### DISCUSSION TITLE
Always online

### DISCUSSION THREAD

**Discussion by 12nick12** [chatwoot/chatwoot#6064] [Q&A] [answered]
With having the mobile app configured and emails configured is there anyway to have the chat always show as online since I'll get an email once someone starts a chat. A push notification would be best, but I know that can get complicated so emails work good enough for that. Once we see the email come in we can open the app and start chatting. 

Thanks for the awesome app.

**Comment by pranavrajs** [comment, marked as answer]
@12nick12 Moved this to the discussions as it is not a bug or feature request.

1. Please set up the business hours and set it for all day to make sure that you are online all the time. 
2. You should get a push notification if a new conversation is created. Please check if you have enabled all the notifications.

**Comment by 12nick12** [comment]
Awesome. Thank you so much.

**Comment by 12nick12** [comment]
I'll test and verify.

**Comment by 12nick12** [comment]
That worked. Thank you.
